### ISSUE TITLE
About YOLONAS License

### ISSUE THREAD

**Issue by fcakyon** [Deci-AI/super-gradients#894]
Hello,

As far as I understood, YOLONAS license prevents the commercial use of any forks of the repo with custom architectural changes. It also prevents the commercial use of any custom finetuned model. Am I right on that?

https://github.com/Deci-AI/super-gradients/blob/1c67ce369b53daf162a91faacb24f3501e7b84ad/LICENSE.YOLONAS.md

**Comment by ofrimasad** [comment]
Hi @fcakyon .
The license does not limit any fork or code use of any kind. 
You can train the model *from scratch* without using the pre-trained weight without any limitations. 
Only if you are using the pre-trained weights you are bounded to the non-commercial license. 
If you fine-tune your model based on the pte-trained weight then you are bounded to the non-commercial license.

**Comment by NeovMs** [comment]
If we use the pre-trained models for knowledge distillation to pre-train the Yolo-NAS from scratch, we're not strictly using the weights. In this case, can we get round the licence ?

**Comment by blacklightpy** [comment]
In that case, it would be the same as the licensing of AI art. That is, free unless the license blocks it.
